src/handles/handle.rs

```diff
@@ -59,14 +59,23 @@ macro_rules! add_constant_handle {
 /// Object handle module
 ///
 pub mod object {
-    use crate::tss2_esys::{ESYS_TR_NONE, ESYS_TR_PASSWORD};
+    use crate::tss2_esys::{
+        ESYS_TR_NONE, ESYS_TR_PASSWORD, ESYS_TR_RH_ENDORSEMENT, ESYS_TR_RH_LOCKOUT,
+        ESYS_TR_RH_NULL, ESYS_TR_RH_OWNER, ESYS_TR_RH_PLATFORM, ESYS_TR_RH_PLATFORM_NV,
+    };
     impl_basic_handle!(
         /// General handle type.
         ObjectHandle
     );
 
     add_constant_handle!(ObjectHandle, PasswordHandle, ESYS_TR_PASSWORD);
     add_constant_handle!(ObjectHandle, NoneHandle, ESYS_TR_NONE);
+    add_constant_handle!(ObjectHandle, OwnerHandle, ESYS_TR_RH_OWNER);
+    add_constant_handle!(ObjectHandle, LockoutHandle, ESYS_TR_RH_LOCKOUT);
+    add_constant_handle!(ObjectHandle, EndorsementHandle, ESYS_TR_RH_ENDORSEMENT);
+    add_constant_handle!(ObjectHandle, PlatformHandle, ESYS_TR_RH_PLATFORM);
+    add_constant_handle!(ObjectHandle, PlatformNvHandle, ESYS_TR_RH_PLATFORM_NV);
+    add_constant_handle!(ObjectHandle, NullHandle, ESYS_TR_RH_NULL);
```

Comment:
Should these be named like the other ones, i.e. `Password` or `PasswordSession`, dropping the `Handle`?

Reply:
What other ones? ```const``` handles for both the different Esys handles and Tpm handles have the Handle-suffix.

Reply:
Indeed, I confused it with the types in `resource_handle.rs` - `HierarchyAuth`, `Enables` etc.

That being said, you choose which version you prefer - we've tended to exclude the suffix/prefix if it's already included in the type name.

Reply:
Added a commit where I removed all the Handle suffixes.